spec/system/user_interacts_with_pdf_viewer_spec.rb

```diff
@@ -182,6 +182,8 @@
   end
 
   context "when adding a suggestion" do
+    let!(:own_suggestions) { create_list(:participatory_documents_suggestion, 2, suggestable: document, author: user) }
```

Comment:
```suggestion
    let!(:own_suggestions) { create_list(:participatory_documents_suggestion, 2, suggestable: document, author: user) }
    let!(:other_suggestions) { create_list(:participatory_documents_suggestion, 2) }
```
Shoudnt' we make sure that there are more suggestion to export only ours?